charts/fpl-case-service/values.yaml

```diff
@@ -5,20 +5,22 @@ tags:
 java:
   image: hmctspublic.azurecr.io/fpl/case-service:latest
   environment:
-    IDAM_API_URL: https://idam-api.aat.platform.hmcts.net
-    IDAM_S2S_AUTH_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
-    RD_PROFESSIONAL_API_URL: http://rd-professional-api-aat.service.core-compute-aat.internal
-    CORE_CASE_DATA_API_URL: http://ccd-data-store-api-aat.service.core-compute-aat.internal
-    SEND_LETTER_URL: http://rpe-send-letter-service-aat.service.core-compute-aat.internal
-    PAYMENT_API_URL: http://payment-api-aat.service.core-compute-aat.internal
-    FEES_REGISTER_API_URL: http://fees-register-api-aat.service.core-compute-aat.internal
-    DOCUMENT_MANAGEMENT_URL: http://dm-store-aat.service.core-compute-aat.internal
-    CCD_UI_BASE_URL: http://www-ccd.aat.platform.hmcts.net
+    IDAM_CLIENT_ID: fpl_case_service
+    IDAM_CLIENT_REDIRECT_URI: https://fpl-case-service-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal/oauth2/callback
```

Comment:
nice :+1: can we also put keyVaults section here, and remove it from preview and aat files?

Reply:
@pd-dudkiewicz  Done. Moved key vaults as well